solcx/install.py

```diff
@@ -131,9 +154,8 @@ def set_solc_version(version, silent=False, solcx_binary_path=None):
 
 
 def set_solc_version_pragma(pragma_string, silent=False, check_new=False):
-    version = _select_pragma_version(
-        pragma_string, [Version(i[1:]) for i in get_installed_solc_versions()]
-    )
+    version_list = get_installed_solc_versions()
```

Comment:
`get_installed_solc_versions` is still returning string objects, which is causing this function to fail.

Reply:
Sneaky Version object